packages/compass-e2e-tests/helpers/commands/click-visible.ts

```diff
@@ -1,16 +1,30 @@
 import type { CompassBrowser } from '../compass-browser';
 
+interface ClickOptions {
+  scroll?: boolean;
+  screenshot?: string;
+}
+
 export async function clickVisible(
   browser: CompassBrowser,
-  selector: string
+  selector: string,
+  options?: ClickOptions
 ): Promise<void> {
   const displayElement = await browser.$(selector);
-
   await displayElement.waitForDisplayed();
 
   // Clicking a thing that's still animating is unreliable at best.
   await browser.waitForAnimations(selector);
 
+  if (options?.scroll) {
+    const scrollElement = await browser.$(selector);
+    await scrollElement.scrollIntoView();
+    await browser.pause(1000);
+  }
+
   const clickElement = await browser.$(selector);
+  if (options?.screenshot) {
```

Comment:
This turns out to be super handy for testing what's actually going on when clicking. Which seems to be where all the bugs materialise, so I think it is worth keeping around.